src/com/spikes2212/Cameras/CameraController.java

```diff
@@ -0,0 +1,63 @@
+package com.spikes2212.Cameras;
+
+import java.util.Optional;
+
+import com.ni.vision.NIVision.Image;
+
+import edu.wpi.first.wpilibj.vision.USBCamera;
+
+public class CameraController {
+	public static int resolutionWidth = 100, resolutionHeight = 100;

```

Comment:
Make final and give reasonable values, also add a constructor in which you can specify those